juspay/hyperswitch-control-center
# chore: profile name validations on edit profile

## Type of Change




## Description

- profile name form validation  in business profile - edit profile name
- validations work same as profile/merchant name validation in home page

![image](https://github.com/user-attachments/assets/ca2ff41e-98af-4711-80ba-bda91b4a3e4f)


## Motivation and Context



## How did you test it?

1. go to settings module
2. go to business profiles
3. click on edit icon
4. try inputting profile name 
5. error appears as per required name validations

## Where to test it?


## Checklist
